### DISCUSSION TITLE
:LazyExtras has some plugins that are not managed by LazyExtras

### DISCUSSION THREAD

**Discussion by StevenXL** [LazyVim/LazyVim#4304] [Q&A] [answered]
I am looking at the floating window that pops up when I type `:LazyExtras`. I see that some plugins are not being managed by LazyExtras (screenshot below). I'm trying to get these managed by LazyExtras, but failing to do so. 

I deleted all files in `plugins`, leaving only `example.lua`, but it still shows some things not managed by LazyExtras:

![image](https://github.com/user-attachments/assets/dc74b43d-e56f-4fa4-9660-04b7bd059de0)

I also do not understand why some `extras` are shown in one color and others in another color, regardless of whether they are "Managed by LazyExtras" or not.

**Comment by dpetka2001** [comment, marked as answer]
You have probably `imported` them in your `~/.config/nvim/lua/config/lazy.lua` file. Only the Extras that you enable via the `LazyExtras` UI by pressing `x` are managed by `LazyExtras`.

**Reply by StevenXL** [reply]
Thank you. That was totally it. Is there any difference between enabling via `lazy.lua` instead of the LazyExtras UI?

**Reply by StevenXL** [reply]
> Thank you. That was totally it. Is there any difference between enabling via `lazy.lua` instead of the LazyExtras UI?

Looks like things got added to `lazyvim.json` and removed from `lazy.lua`. I think I prefer it `lazyvim.json`. Thank you for the help!

**Reply by dpetka2001** [reply]
It's recommended to use the `LazyExtras` UI because that also takes care of the order the plugins will load. Some plugins need to load before others. If you do it in `lua/config/lazy.lua`, then you have to take care of the order yourself because they will load in the way they are parsed.
